src/common/module/yoecldp.F90

```diff
@@ -366,6 +372,11 @@ SUBROUTINE YRECLDP_LOAD_PARAMETERS()
     ! The last two are actually arrays, so treat them as fields
     CALL LOAD_ARRAY('YRECLDP_RBETA', 1, 101, 101, 101, YRECLDP%RBETA(0:100))
     CALL LOAD_ARRAY('YRECLDP_RBETAP1', 1, 101, 101, 101, YRECLDP%RBETAP1(0:100))
+    YRECLDP%NCLDQL = NCLDQL 
```

Comment:
Not too keen on this tbh. We know that actually using these from the Fortran side will make the code slow, so there's no obvious gain here, but increased risk someone's going to fall for that.